### PR TITLE
Thu/visual bugs

### PR DESCRIPTION
**Pull request by thumn**
## Visual Bugs
Fixed some visual bugs: applying min-width to page body, making checkbox backgrounds correspond to page color scheme when checked, changing Edit Artist Profile theme to indigo, renaming "History and Theory," and modifying link colors to match artist theme in Artist Snapshots.

### Related PRs
None.

### Migrations
None.

### Tests Performed, Edge Cases
None.

### Screenshots
Screen Shot 2019-03-18 at 7 37 25 PM
Screen Shot 2019-03-18 at 7 37 34 PM
Screen Shot 2019-03-18 at 7 37 49 PM

CC: @by-co

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
  body {
     background-color: $snow;
+    min-width: 1500px;
```
Comment (by-co) [inline app/assets/stylesheets/application.css.scss]: Hmm, let's bump this down to `1200px`? https://css-tricks.com/snippets/css/media-queries-for-standard-devices/

File: app/assets/stylesheets/components/snapshot.scss [hunk 2]
```diff
@@ -6,4 +6,8 @@
   .snapshot-nav:hover {
     @extend .pointer;
   }
+
+  .link-magenta {
```
Comment (by-co) [inline app/assets/stylesheets/components/snapshot.scss]: Looks like we're not using this, can we delete the class :3

File: app/javascript/components/helpers/UserSnapshot.jsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ class UserSnapshot extends React.Component {
         </Touchable>
         <div className="snapshot-content">
           <Touchable onPress={() => this.props.navigate()}>
-            <h5 className="snapshot-nav">{this.props.name}</h5>
+            <h5 className={`snapshot-nav link-${this.props.color}`}>{this.props.name}</h5>
```
Comment (by-co) [inline app/javascript/components/helpers/UserSnapshot.jsx]: 🙌

File: app/javascript/components/works/Filters.jsx [hunk 4]
```diff
@@ -69,15 +70,16 @@ class Filters extends PureComponent {
                 <div className="checkbox-container">
                   {Object.keys(filters[type]).map(item => (
                     <div className="mb2 checkbox-item" key={item}>
-                      <label for={`checkbox-${item}`}>
+                      <label className = "ttc dib flex" for={`checkbox-${item}`}>
```
Comment (by-co) [inline app/javascript/components/works/Filters.jsx]: Screen Shot 2019-03-18 at 8 43 33 PM

Hmm this is still a little wonky for me for some reason :( 

@ethanlee16 is it showing up fine for you?

Comment (by-co) [reply app/javascript/components/works/Filters.jsx]: @thumn I think the solution is to give the text a width manually:
```
<span className="filter-item">{convertSnakeCase(item)}</span>
```

```
.filter-item {
    width: 146px;
}
```
